lamby/controllers/models.py

```diff
@@ -1,13 +1,34 @@
-from flask import Blueprint, render_template
+from flask import Blueprint, flash, redirect, render_template, url_for
 
 from lamby.filestore import fs
-
+from lamby.models.commit import Commit
+from lamby.forms.netron import HeadForm
 models_blueprint = Blueprint('models', __name__)
 
 
 @models_blueprint.route('/<string:project_id>/<string:commit_id>')
 def model(project_id, commit_id):
+    commit = Commit.query.filter_by(id=commit_id)
     return render_template(
         'netron.jinja',
-        object_link=fs.get_link(f'{project_id}/{commit_id}')
+        object_link=fs.get_link(f'{project_id}/{commit_id}'),
+        commits=Commit.query.filter_by(
+            project_id=project_id, filename=commit[0].filename),
+        project_id=project_id,
+        current_commit=commit_id,
+        reset_head_form=HeadForm()
     )
+
+@models_blueprint.route('resethead/<string:project_id>/<string:commit_id>')
```

Comment:
nit: change head might be better naming

Reply:
@anoopj08 can you make this change so ammu can approve it.